boards/xtensa/intel_adsp_cavs18/doc/generic_intel_adsp.rst

```diff
@@ -46,7 +67,18 @@ Build and run the tests
 
 .. code-block:: console
 
-   west flash --remote-host [remote hostname or ip addr]
+   west flash --remote-host {host}:{port} \
+              --pty {host}:{port}\
+Ex.
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/issues/48215#issuecomment-1193097630

Reply:
Thanks for your help. Got it fixed.